dtm0/drlink.c

```diff
@@ -31,6 +31,12 @@
 #include "rpc/rpc.h"                 /* m0_rpc_item_post */
 #include "rpc/rpc_machine.h"         /* m0_rpc_machine */
 #include "rpc/rpc_opcodes.h"         /* M0_DTM0_{RLINK,REQ}_OPCODE */
+#include "lib/string.h"              /* m0_streq */
+
+enum {
+	DRLINK_CONNECT_TIMEOUT_SEC = 1,
+	DRLINK_DISCONN_TIMEOUT_SEC = DRLINK_CONNECT_TIMEOUT_SEC,
```

Comment:
I am not sure if this time out value is the best value.
We have a lot of time out values in the Motr system.
One second seems to be a very small timeout value. What if the connect can not be established in a busy system?

Reply:
These timeouts will be reviewed after we fix RPC connection issue.

Reply:
Updated the comment in code.

Reply:
Fixed/merged.